lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextRuntime.cs

```diff
@@ -331,7 +332,7 @@ public void DeliverTaskMessage(byte[] message)
             }
         }
 
-        public void HandleContextMessaage(byte[] mesage)
+        public void HandleContextMessage(byte[] mesage)
```

Comment:
Oh, this is `public` method. Please keep the legacy function with deprecation annotation, too.